.vscode/settings.json

```diff
@@ -1,5 +1,4 @@
 {
-    "cSpell.words": [
```

Comment:
Hmm.. actually, this file should not be a part of the repository, it landed here by mistake (when migrating documentation from other repo)

VS Code settings is mostly about personal settings (and it also implies VS Code as an editor choice, which also should be up to developer)

I've removed it from the repo. You can ensure it doesn't end in your git diff by updating gitignore global rules (see https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

Reply:
Yeah makes sense 👍